docs/organization/dynamic-sampling/index.mdx

```diff
@@ -1,18 +1,54 @@
 ---
-title: Retention Priorities
-sidebar_order: 130
-description: "Learn how Sentry determines retention priorities and how to update them."
+title: Dynamic Sampling
+sidebar_order: 30
+description: "Learn how to make your billing quota go farther with Sentry's Dynamic Sampling."
 ---
 
+## Overview
+
+High-volume projects that generate many events can quickly consume a large portion of your billing quota, leaving less room for other projects and decreasing visibility across them.
+
+With high-volume projects, you'll often see many similar events, which can be redundant and less helpful for debugging. But with Dynamic Sampling, you can maximize your quota by adjusting sample rates based on each project’s event volume. The sample rate is a percentage from 0% (no events passed) to 100% (all events passed), that can be tailored to each project’s needs.
+
+When you use Dynamic Sampling, projects in your organization are sampled based on their recent event volumes. High-volume projects are sampled at lower rates while lower-volume projects are sampled at higher rates or not sampled at all, based on your recent event data. This makes it possible for each project to retain a meaningful set of key events, improving visibility into all projects regardless of their size.
+
+![The dynamic sampling mode and target sample rate setting.](./img/mode-and-target-sample-rate.png)
+
+## Modes
+
+You can choose between two Dynamic Sampling modes for your organization: manual or automatic.
+
+    * **`automatic`** (default): In automatic mode, the sample rate for each project is set automatically based on relative project volume, so events fit within your organization’s billing quota. This boosts visibility into lower-volume projects. To account for any changes in event volume, sample rates are recalibrated on a regular basis. Automatic mode overrides any manually-set project sample rates. 
+    
+    * **`manual`**: If you want to set the exact sample rate for each project in your organization yourself, choose manual mode. Note, that in this mode, there is no automatic recalibration.
+
+
+[]: # (TODO: Rename AM3 with the actual user-facing name.)
+
 <Note>
 
-Retention priorities are only available on the latest version of our plans, starting from 1M reserved transactions. Anyone in your org with owner or billing-level permissions can update your plan by going to Settings > Subscription, and clicking the "Manage Subscription" button.
+Please note that these Dynamic Sampling modes are only available for customers on the AM3 platform.
```

Comment:
Should we call out that on AM2 there's also DS, the sample rate is chosen by the system based on transaction volume (1M+), and you still have controls for the sampling priorities?